<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="1" failures="0" hostname="beck-PC" name="pytest" skipped="0" tests="1" time="0.942" timestamp="2020-04-19T16:20:06.189900"><testcase classname="pytest" name="internal" time="0.000"><error message="internal error">Traceback (most recent call last):
  File &quot;D:\program\Python37\lib\site-packages\_pytest\main.py&quot;, line 191, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;D:\program\Python37\lib\site-packages\_pytest\main.py&quot;, line 235, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\hooks.py&quot;, line 289, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\manager.py&quot;, line 87, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\manager.py&quot;, line 81, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get(&quot;firstresult&quot;) if hook.spec else False,
  File &quot;D:\program\Python37\lib\site-packages\pluggy\callers.py&quot;, line 208, in _multicall
    return outcome.get_result()
  File &quot;D:\program\Python37\lib\site-packages\pluggy\callers.py&quot;, line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File &quot;D:\program\Python37\lib\site-packages\pluggy\callers.py&quot;, line 187, in _multicall
    res = hook_impl.function(*args)
  File &quot;D:\program\Python37\lib\site-packages\_pytest\main.py&quot;, line 256, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\hooks.py&quot;, line 289, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\manager.py&quot;, line 87, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\manager.py&quot;, line 81, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get(&quot;firstresult&quot;) if hook.spec else False,
  File &quot;D:\program\Python37\lib\site-packages\pluggy\callers.py&quot;, line 208, in _multicall
    return outcome.get_result()
  File &quot;D:\program\Python37\lib\site-packages\pluggy\callers.py&quot;, line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File &quot;D:\program\Python37\lib\site-packages\pluggy\callers.py&quot;, line 187, in _multicall
    res = hook_impl.function(*args)
  File &quot;D:\program\Python37\lib\site-packages\_pytest\runner.py&quot;, line 72, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;D:\program\Python37\lib\site-packages\_pytest\runner.py&quot;, line 81, in runtestprotocol
    rep = call_and_report(item, &quot;setup&quot;, log)
  File &quot;D:\program\Python37\lib\site-packages\_pytest\runner.py&quot;, line 169, in call_and_report
    report = hook.pytest_runtest_makereport(item=item, call=call)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\hooks.py&quot;, line 289, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\manager.py&quot;, line 87, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File &quot;D:\program\Python37\lib\site-packages\pluggy\manager.py&quot;, line 81, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get(&quot;firstresult&quot;) if hook.spec else False,
  File &quot;D:\program\Python37\lib\site-packages\pluggy\callers.py&quot;, line 203, in _multicall
    gen.send(outcome)
  File &quot;e:\virtual_workshop\app_autotest\test_cases\conftest.py&quot;, line 55, in pytest_runtest_makereport
    screen_img = _capture_screenshot()
  File &quot;e:\virtual_workshop\app_autotest\test_cases\conftest.py&quot;, line 65, in _capture_screenshot
    return driver.get_screenshot_as_base64()
AttributeError: &apos;NoneType&apos; object has no attribute &apos;get_screenshot_as_base64&apos;</error></testcase></testsuite></testsuites>